source/_redirects

```diff
@@ -6,75 +6,75 @@
+/developers/asyncio_101 https://developers.home-assistant.io/docs/asyncio_101/
+/developers/asyncio_categorizing_functions https://developers.home-assistant.io/docs/asyncio_categorizing_functions/
+/developers/asyncio_misc https://developers.home-assistant.io/docs/asyncio_index/
+/developers/asyncio_working_with_async https://developers.home-assistant.io/docs/asyncio_working_with_async/
+/developers/asyncio https://developers.home-assistant.io/docs/asyncio_index/
+/developers/code_review_component https://developers.home-assistant.io/docs/creating_component_code_review/
+/developers/code_review_platform https://developers.home-assistant.io/docs/creating_platform_code_review/
+/developers/component_deps_and_reqs https://developers.home-assistant.io/docs/creating_component_deps_and_reqs/
+/developers/component_discovery https://developers.home-assistant.io/docs/creating_component_discovery/
+/developers/component_events https://developers.home-assistant.io/docs/creating_component_events/
+/developers/component_generic_discovery https://developers.home-assistant.io/docs/creating_component_generic_discovery/
+/developers/component_loading https://developers.home-assistant.io/docs/creating_component_loading/
+/developers/component_states https://developers.home-assistant.io/docs/creating_component_states/
+/developers/component_visibility https://developers.home-assistant.io/docs/creating_component_index/
+/developers/creating_components https://developers.home-assistant.io/docs/creating_component_index/
+/developers/development_101 https://developers.home-assistant.io/docs/dev_101_index/
+/developers/development_catching_up https://developers.home-assistant.io/docs/development_catching_up/
+/developers/development_checklist https://developers.home-assistant.io/docs/development_checklist/
+/developers/development_config https://developers.home-assistant.io/docs/dev_101_config/
+/developers/development_environment https://developers.home-assistant.io/docs/development_environment/
+/developers/development_events https://developers.home-assistant.io/docs/dev_101_events/
+/developers/development_guidelines https://developers.home-assistant.io/docs/development_guidelines/
+/developers/development_hass_object https://developers.home-assistant.io/docs/dev_101_hass/
+/developers/development_services https://developers.home-assistant.io/docs/dev_101_services/
+/developers/development_states https://developers.home-assistant.io/docs/dev_101_states/
+/developers/development_submitting https://developers.home-assistant.io/docs/development_submitting/
+/developers/development_testing https://developers.home-assistant.io/docs/development_testing/
+/developers/development_validation https://developers.home-assistant.io/docs/development_validation/
+/developers/development https://developers.home-assistant.io/docs/development_index/
+/developers/documentation/create_page https://developers.home-assistant.io/docs/documentation_create_page/
```

Comment:
Instead of going for the redirects on developers.ha.io. We should redirect directly to the target from ha.io.

E.g., 

```suggestion
/developers/documentation/create_page https://developers.home-assistant.io/docs/documenting/create-page/
```

Reply:
I agree, however, the move on the developer documentation is still ongoing progress.

I suggest merging this, as this solves the problems caused by the initial move to Docusaurus 2 a while back. Redirects will be able to handle this for now.

After the move upstream has been completed, we should revisit these URLs.